modules/network-observability-operator-install.adoc

```diff
@@ -10,6 +10,7 @@ You can install the Network Observability Operator using the {product-title} web
 .Prerequisites
 
 * Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6].
+* One of the following supported architectures is required: `amd64`, `ppc64le`, or `arm64`.
```

Comment:
@skrthomas Thinking we should add the caveat for IBM P8 here?

Reply:
+1 this is the right place to mention `rhel9`
https://issues.redhat.com/browse/NETOBSERV-1067

Reply:
@jpinsonneau I went ahead and added MT Loki 5.7 prereq info in this PR too, since this PR is shaping to be a bit of a prereq catchall.

Reply:
@nathan-weinberg I don't see this mentioned specifically in https://issues.redhat.com/browse/NETOBSERV-576...curious where you're seeing it or if it was previously supported but undocumented?

Reply:
@nathan-weinberg I don't think we need to add `s390x` per Joel's response to your question about it in the team channel. If everything else look OK, can you approve from QE side?